doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -217,6 +217,11 @@ numbers (see :ref:`datatypes`).
 Sorts
 -----
 
+.. index::
+   single: Set
```

Comment:
Adding this index entry reveals a bug:
The next index entry `Set (cmd)` now appears as `(cmd)`.
On the other hand, if I put `Set (sort)` here, `Set (cmd)` is displayed normally but `Set (sort)` is displayed as `(sort)`.
I have observed the same bug with the two indices for `term` (which I added in this PR).

Reply:
I investigated this, and I'm not sure it's a bug :)

It's definitely a rendering issue, though; it currently looks confusing.  It comes from here: https://github.com/sphinx-doc/sphinx/blob/93837bb01ecb185d8ee64b1f794f010c510b32e1/sphinx/environment/adapters/indexentries.py#L114

We could either monkey-patch or open an issue to make this customizable.